index.js

```diff
@@ -18,22 +17,22 @@ module.exports = class ngAnnotateCompiler {
   compile(file) {
     if (this.isIgnored(file.path)) return Promise.resolve(file)
     const options = withInFile(this.config, file.path)
-    const {src, map = file.map, errors} = ngAnnotate(file.data, options)
```

Comment:
This means `map` will be assigned to the `map` property of the returned value, or if that's `undefined`, to `file.map`. Your version always uses `file.map`. Can you fix it?